refactor/src/Cage.py

```diff
@@ -557,19 +659,34 @@ def update_distrib_discrete_add(self, distrib_delta, distrib):
                 distrib[geno] = 0
             distrib[geno] += distrib_delta[geno]
 
-    def update_distrib_discrete_subtract(self, distrib_delta, distrib):
+    @staticmethod
+    def update_distrib_discrete_subtract(distrib_delta, distrib):
         for geno in distrib:
             if geno in distrib_delta:
                 distrib[geno] -= distrib_delta[geno]
             if distrib[geno] < 0:
                 distrib[geno] = 0
 
+    @staticmethod
+    def multiply_distrib(distrib: dict, population: int):
```

Comment:
A bit of a naive question, but is there no simpler way of doing this?

Reply:
the problem is not in the multiplication step per se, but in making sure casting to integers preserves a few invariants.